tests/test_backend.py

```diff
@@ -177,11 +177,8 @@ def expect_fail(test_case_path):  # type: (str) -> None
 expect_fail('OnnxBackendNodeModelTest.test_averagepool_2d_ceil_cpu')
 
 # Modulus - NGONNX-527
+# fmod=0 is not supported
```

Comment:
Please create tickets for the failing tests and add the ticket numbers by the disabled tests.

Reply:
I updated existing `Modulus` operator ticket